lib/io.js

```diff
@@ -26,14 +27,15 @@ module.exports = class IO {
     this.onresponse = onresponse
     this.ontimeout = ontimeout
 
+    this._udx = new UDX()
```

Comment:
doesn't this need to see if udx is passed in the options?

Reply:
It does indeed!